app/models/metric/purging.rb

```diff
@@ -164,16 +140,11 @@ def self.purge_in_batches(scope, window, total = 0, total_limit = nil)
         break if count == 0
         total += count
 
-        if scope.klass == MetricRollup
-          count_tag_values = purge_associated_records(scope.name, batch_ids)
-          total_tag_values += count_tag_values
-        end
-
```

Comment:
I didn't see this before - so nice. 
This method is now standard `purge_in_batches`  +💯 

Besides `interval`, this whole file is basically standard `PurgingMixin`.